package Protocol;
message S2CRobotLoginRsp
{
	required bool login_result = 1;
}